test/integration/068_partial_parsing_tests/test-files/env_var_metrics.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+
+metrics:
+
+  - model: "ref('people')"
+    name: number_of_people
+    description: Total count of people
+    label: "Number of people"
+    type: count
+    sql: "*"
```

Comment:
Should I anticipate this working for non-`meta` fields as well?

I set an env var for `sql` here like:
```yml
    sql: '{{ env_var("ENV_VAR_SQL", "*") }}'
```
Then:
```
$ dbt ls --resource-type metric --output json --output-keys name,sql
{"name": "number_of_people", "sql": "*"}
```
Looks good. But it doesn't seem to update when the env var changes:
```
$ ENV_VAR_SQL=person_id dbt ls --resource-type metric --
{"name": "number_of_people", "sql": "*"}
```

Reply:
Strange. The metrics nodes are only rendered in one place, so I assumed just checking one would be enough. I'll check this out.

Reply:
I think this should be working now.